### DISCUSSION TITLE
3 questions and a suggestion

### DISCUSSION THREAD

**Discussion by JLNjolo** [rrirower/DP2SceneGraphWizard#89] [Q&A] [answered]
1. Why is my logo now much smaller?
2. Why is the Title not showing?
3. Why does the * bring nothing up on the Homescreen?
Suggestion: Make the font color selectable in your software because I am not a brightscript expert and your Wiki info is like a foreign language to me. The dark blue default doesn't work with a black background, which many channels use. Otherwise white would work better, since many channels also use it.
Thanks.

**Comment by rrirower** [comment, marked as answer]
1. It's difficult to know what you're talking about without seeing your screen.  A screen image would help.
2. Again, I'm not sure what you're talking about.  What title?
3. If you read the release notes for the version you downloaded, you'll see that the * is now linked to a Search function.  You need to enable that function to use the '*' key.

Either way, you can manage the look and feel of the channel by setting the optional manifest attributes.  The wiki contains information on how to do that.

As per your suggestion, you don't need to be a BrightScript expert to change the font or colors.  You simply need to change the color code in the _fontsupport.brs_ file for the font you want to update.  For example, this is an entry in that file:

```
"GothamBoldDarkBlue": {
            "fontUri": "pkg:/fonts/Gotham-Bold.otf"
            "fontSize": 36
            "color" : "#00008b"
        }
```

the color code is '00008b' above.  Using an [RGB color code table](https://www.w3schools.com/colors/colors_hex.asp), you can find a different color, for example, 'Ivory' (#fffff0), and change '00008b' to 'fffff0'.

I will update the Wiki to make this a little clearer.

**Reply by JLNjolo** [reply]
That works. Thanks. One more question: Can I change the screen thumbnail highlight color? If so, how? It is the border around the thumbnail when searching through videos on the screen.

**Reply by rrirower** [reply]
If you're referring to the white border around a thumbnail while scanning through content, it's an image file.  It's in the image folder and is called "focus.9.png".  You'll need to replace it with a file the same size, etc.

**Comment by JLNjolo** [comment]
That worked, but not on the homescreen images which is probably a Roku thing.[cid:6a9c88f7-e682-4394-8391-aeb690cf4e9a]
________________________________
